benefits/settings.py

```diff
@@ -213,7 +213,6 @@
 
 CSP_SCRIPT_SRC = [
     "'unsafe-inline'",
-    "https://california.azureedge.net/cdt/statetemplate/",
```

Comment:
Perfect! Since this JS file we are removing is responsible for loading State Template fonts, and we don't need those anymore, can we also remove the State Template entry from `CSP_FONT_SRC` please?